### ISSUE TITLE
[FEAT] Delete added website with shortcut

### ISSUE THREAD

**Issue by toprak** [willbchang/chrome-read-later#222]
**Is your feature request related to a problem? Please describe.**
it just frustrating click to extension's icon and delete
**Describe the solution you'd like**
It would be great to add a separate shortcut to delete the urls we open from the reading list, just as we can delete them with alt shift s.

**Comment by willbchang** [comment]
> it just frustrating click to extension's icon and delete
Yes, I understand opening and deleting is frustrating. That's why I designed this extension, it will remove the reading item after opening it from the reading list. I'm just curious, did you often removing items from the reading list rather than opening it? I'd like to hear the how do you usually use it and we may improve this extension based on that.

> It would be great to add a separate shortcut to delete the urls we open from the reading list, just as we can delete them with alt shift s.
I didn't get it, could you help me to stay on the same page?

Did yo mean:
A. Undo the adding to the reading list action without opening the reading list.
B. Delete the opened reading item from the reading list.
C. Delete the reading litem without opening the reading list.
D. Open the first reading item and delete it without opening the reading list.

I appreciate your feedback.

**Comment by toprak** [comment]
@willbchang I'm a bit sleep deprived, sorry I didn't explain it properly, it's a good thing to delete a url from the list when you open it, but when lazy people like me open that tab and then unwittingly deal with other things, we can close the browser without reading that tab, and that tab is gone from the list, so there is a problem. My suggestion is that this can also be added as an option, the url we select from the list should not be deleted when we open the url, it should be deleted from the list with the shortcut we specially assign while the url is open.

**Comment by willbchang** [comment]
@toprak I totally get it! Make it as an option is acceptable, I'll add this feature in the next release.

By the way did you notice the history mode? Opening url from it won't be deleted. The history mode stays locally.
<img width="264" alt="image" src="https://github.com/willbchang/chrome-read-later/assets/14329786/a902ba38-e41b-4792-900a-9df9245fd07f">
